dispatchers.go

```diff
@@ -253,15 +277,40 @@ func dispatchCopy(b *Builder, args []string, attributes map[string]bool, flagArg
 			if from == "" {
 				return fmt.Errorf("no value specified for --from=")
 			}
+		case arg == "--link", arg == "--link=true":
+			link = true
+		case arg == "--link=false":
+			link = false
+		case arg == "--parents", arg == "--parents=true":
+			parents = true
+		case arg == "--parents=false":
+			parents = false
+		case strings.HasPrefix(arg, "--exclude="):
+			exclude := strings.TrimPrefix(arg, "--exclude=")
+			if exclude == "" {
+				return fmt.Errorf("no value specified for --exclude=")
+			}
+			excludes = append(excludes, exclude)
 		default:
-			return fmt.Errorf("COPY only supports the --chmod=<permissions> --chown=<uid:gid> and the --from=<image|stage> flags")
+			return fmt.Errorf("COPY only supports the --chmod=<permissions>, --chown=<uid:gid>, --from=<image|stage>, --link, --parents, and --exclude=<pattern> flags")
 		}
 	}
 	files, err := processHereDocs(buildkitcommand.Copy, original, heredocs, userArgs)
 	if err != nil {
 		return err
 	}
-	b.PendingCopies = append(b.PendingCopies, Copy{From: from, Src: args[0:last], Dest: dest, Download: false, Chown: chown, Chmod: chmod, Files: files})
+	b.PendingCopies = append(b.PendingCopies, Copy{
+		From:     from,
+		Src:      args[0:last],
+		Dest:     dest,
+		Download: false,
+		Chown:    chown,
+		Chmod:    chmod,
+		Files:    files,
+		Link:     link,
+		Parents:  parents,
+		Excludes: excludes,
```

Comment:
I'm assuming the comma at the end of the last argument is OK, it wasn't there prior.

Reply:
It's required when the multi-line literal isn't closed on the same line.